packages/bpk-component-map/src/BpkMap.js

```diff
@@ -18,11 +18,141 @@
 
 /* @flow */
 
-import React from 'react';
+import React, { type Node } from 'react';
+import PropTypes from 'prop-types';
 import { withGoogleMap, GoogleMap } from 'react-google-maps';
 
-const BpkMap = withGoogleMap(({ mapRef, ...rest }) => (
-  <GoogleMap ref={mapRef} {...rest} />
-));
+type MapRef = ?{
+  getBounds: () => Object,
+  getCenter: () => Object,
+  getZoom: () => number,
+  fitBounds: ({
+    south: number,
+    west: number,
+    north: number,
+    east: number,
+  }) => void,
+};
+
+type Props = {
+  scrollEnabled: boolean,
+  zoom: number,
+  zoomEnabled: boolean,
+  bounds: ?{
+    south: number,
+    west: number,
+    north: number,
+    east: number,
+  },
+  center: ?{
```

Comment:
This object is going to be so common that it's probably a good idea to create a distinc type for it instead of repeating it in many places

Reply:
Good shout - I've done that now.